sdks/python/apache_beam/transforms/ptransform.py

```diff
@@ -374,6 +374,12 @@ def __ror__(self, left, label=None):
       p = pipeline.Pipeline(
           'DirectRunner', PipelineOptions(sys.argv))
     else:
+      if any(getattr(p, 'is_ephemeral', False) for p in pipelines):
```

Comment:
should this be `True`?

Reply:
No, this should be False, which is the correct value if it's not present.